apps/platform-app/src/auth.ts

```diff
@@ -92,6 +115,9 @@ export const {
       session.user.githubUsername = foundUser.githubDetails
         ? foundUser.githubDetails.username
         : ''
+      session.user.linkedinUsername = foundUser.linkedinDetails
+        ? foundUser.linkedinDetails.username
+        : ''
 
       return session
     },
```

Comment:
**nitpick**: Remove console.log statements in production code to avoid unnecessary clutter in the console output.

Reply:
no console.log here. the nearest one is line 133